CliClient/locales/zh_CN.po

```diff
@@ -636,9 +636,9 @@ msgid ""
 "to list tags associated with [note]. The command `tag list` can be used to "
 "list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> 可以是“add”、“remove”、“list” 或者 “notetags”， 用于从 [note] "
-"中赋值或删除 [tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出"
-"所有的标签（对于过长选项请使用 -l 参数）。"
+"<tag-command> 可以是“add”、“remove”、“list” 或者 “notetags”，用于从 [note] 中"
+"赋值或删除 [tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出所"
+"有的标签（对于过长选项请使用 -l 参数）。"
```

Comment:
After `"notetags"` and `[tag]` you use adifferent comma. Is that on purpose?

Reply:
yes